src/CardTitle.js

```diff
@@ -12,12 +12,8 @@ const propTypes = {
   tag: tagPropType,
 };
 
-const defaultProps = {
-  tag: 'div',
-};
-
 function CardTitle(props) {
-  const { className, cssModule, tag: Tag, ...attributes } = props;
+  const { className, cssModule, tag: Tag = 'div', ...attributes } = props;
```

Comment:
Since this is a title, I think it'd be better to use `h5` or something here.

Reply:
My intention is to not change any behaviour with this pr. The original default value is `div` which I have preserved.